fedimint-api/src/config.rs

```diff
@@ -20,3 +21,10 @@ pub trait GenerateConfig: Sized {
     /// Asserts that the public keys in the config are and panics otherwise (no way to recover)
     fn validate_config(&self, identity: &PeerId);
 }
+
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct BitcoindRpcCfg {
+    pub btc_rpc_address: String,
+    pub btc_rpc_user: String,
+    pub btc_rpc_pass: String,
```

Comment:
Since we are touching it, would [`Auth`](https://docs.rs/bitcoincore-rpc/0.16.0/bitcoincore_rpc/enum.Auth.html) make sense here? Probably out of scope though,

Reply:
It probably would, but yeah - some other time. :)